rules/pom.xml

```diff
@@ -11,7 +11,11 @@
   <packaging>pom</packaging>
 
   <modules>
+    <!--
```

Comment:
I would be keener on removing it right away to avoid maintaining of a dead code, or keep it here and mark as deprecated.